doc/02-Backends.md

```diff
@@ -0,0 +1,397 @@
+## Backends
+
+Once your program is compiled with Verificarlo, it can be instrumented with
+different floating-point backends.
+At least one backend must be selected when running your application,
+
+```bash
+   $ verificarlo-c *.c -o program
+   $ ./program
+   program: VFC_BACKENDS is empty, at least one backend should be provided
+```
+
+Backends are distributed as dynamic libraries. They are loaded with the
+environment variable `VFC_BACKENDS`.
+
+```bash
+   $ VFC_BACKENDS="libinterflop_mca.so" ./program
+```
+
+Multiple backends can be loaded at the same time; they will be chained in the
+order of appearance in the `VFC_BACKENDS` variable. They must be separated with
+semi-colons,
+
```

Comment:
add list of links to all backend sections

Reply:
I added the list and links at the beginning of the Backends chapter.